CONTRIBUTING.md

```diff
@@ -2,79 +2,9 @@
 
 The APPUiO Charts project accepts contributions via GitHub pull requests. This document outlines the process to help get your contribution accepted.
 
-## Sign Your Work
```

Comment:
Why are we removing this section?

Reply:
We discussed a few months ago that we don't require DCO anymore, so I removed it for cleanup reasons.